native/src/main/java/io/ballerina/stdlib/protobuf/nativeimpl/AnyTypeCreator.java

```diff
@@ -51,10 +64,16 @@ public class AnyTypeCreator {
     private AnyTypeCreator() {
 
     }
-
     public static BString externGetNameFromRecord(BMap<BString, Object> value) {
-
-        return StringUtils.fromString(value.getType().getName());
+        String name = value.getType().getName();
+        if ("map".equals(name)) {
+            return StringUtils.fromString("google.protobuf.Struct");
+        }
+        String packageName = value.getType().getPackage().getName();
+        if ("Any".equals(name) && "protobuf.types.any".equals(packageName)) {
```

Comment:
better to use constants for "protobuf.types.any"

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-protobuf/pull/176/commits/0b4e304a70313de3a78afb5bc4ffe5ce99f8cd6b